wof/examples/production_configs/wofpy.conf

```diff
@@ -15,5 +15,5 @@ setgid www-data
 env PATH=/home/ubuntu/miniconda/envs/wofpy/bin
 
 # change directory to where the uWSGI configuration file is located then execute uwsgi with the config file
-chdir /var/www/timeseries
-exec uwsgi --ini timeseries.ini
\ No newline at end of file
+chdir /var/www/wofpy
+exec uwsgi --ini wofpy.ini
```

Comment:
Got it! So this means that a user who wants to run **more than one** service would still only run that single line `systemctl start wofpy`?!

Crossing fingers that the answer is yes :smiley: ... I don't see anything in `wofpy.conf`, `wofpy.ini` or `wofpy.service` that suggests they're limited to one wofpy service.